src/swap/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const LOW_LIQUIDITY_ERROR_CODE = 'SWAP_QUOTE_LOW_LIQUIDITY_ERROR';
+export const GENERAL_SWAP_ERROR_CODE = 'SWAP_ERROR';
+export const GENERAL_SWAP_QUOTE_ERROR_CODE = 'SWAP_QUOTE_ERROR';
+export const GENERAL_SWAP_BALANCE_ERROR_CODE = 'SWAP_BALANCE_ERROR';
+export const UNCAUGHT_SWAP_QUOTE_ERROR_CODE = 'UNCAUGHT_SWAP_QUOTE_ERROR';
+export const UNCAUGHT_SWAP_ERROR_CODE = 'UNCAUGHT_SWAP_ERROR';
```

Comment:
Should ERROR_CODE being a post-fix or a pre-fix.
mmmmmm

I know in English it's more correct as post-fix, but in reading code might be not as easy. Let's leave it like this for now. But I have a feeling all const error, might be better as error code pre-fix. 

OR

we can create a mapping system like we are doing for themeing. And just create a `const errorCode = {}` which can be autocomplete when coding.